actioncable/lib/action_cable/subscription_adapter/base.rb

```diff
@@ -0,0 +1,33 @@
+module ActionCable
+  module SubscriptionAdapter
+    class Base
+      attr_reader :logger, :server
+
+      def initialize(server)
+        @server = server
+        @logger = @server.logger
+      end
+
+      def self.require_gems!(*gems)
+        begin
+          gems.each { |gem| require gem }
```

Comment:
WE should use the same version of pg that we use in AR. And we should call `gem 'pg'` like we do in AR.